### ISSUE TITLE
Fix jsx-a11y/no-noninteractive-tabindex warnings raised by jsx-a11y linter.

### ISSUE THREAD

**Issue by yzen** [firefox-devtools/debugger#7544]
To get a list of accessibility warnings raised by jsx-a11y linter run: `yarn lint:jsx-a11y`.

From that list please note all instances that are flagged to have "jsx-a11y/no-noninteractive-tabindex" problem. Each one needs to be addressed individually.

Once all of the instances are resolved make sure to change this setting in `.eslintrc.jsx-a11y` from:
```
"jsx-a11y/no-noninteractive-tabindex": ["warn", {
  "tags": [],
  "roles": ["tabpanel"]
}],
```

to
```
"jsx-a11y/no-noninteractive-tabindex": ["error", {
  "tags": [],
  "roles": ["tabpanel"]
}],
```

**Comment by Garbee** [comment]
[This rule](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-tabindex.md) is about `tabindex` being used on elements that are not interactive by default, so they aren't providing any real benefit being there. I believe this *should* be mostly resolved in fixing #7537 as well. But quite possibly not fully resolved.

**Comment by janodvarko** [comment]
Firefox Developer Tools (including Debugger tool) development is entirely happening in Bugzilla and new issues should be reported here: https://bugzilla.mozilla.org/enter_bug.cgi?product=DevTools&component=Debugger

Also, supporting JSX in the code base is not on our long term roadmap, so this might be invalid report.

Honza
